src/style.css

```diff
@@ -0,0 +1,73 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+  
+@font-face {
+  font-family: "Avenir";
+  font-weight: 700;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+  
+.form__section-big,
+.form__section-small {
+  position: relative;
+  display: block;
+  margin: 20px auto;
+}
+  
+.image {
+  position: absolute;
+}
+  
+.big-image {
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+  
+.small-image {
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+}
+  
+.form__search {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  width: 100%;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  outline: none;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+  
+.big__search {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+  
+.small__search {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
```

Comment:
Don't repeat yourself. Color is the same for both search bars. You can move it to common class